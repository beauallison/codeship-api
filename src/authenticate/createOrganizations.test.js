const createOrganizations = require('./createOrganizations');
const status201 = require('./__fixtures__/status201.json');

describe('authenticate/createOrganizations()', () => {
  it('should create organizations', () =>
    expect(createOrganizations(status201.data.organizations)).toMatchSnapshot());

  it('should return an empty Map on missing data', () =>
    expect(createOrganizations()).toMatchSnapshot());
});
