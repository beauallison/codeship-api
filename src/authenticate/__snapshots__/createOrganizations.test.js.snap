// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authenticate/createOrganizations() should create organizations 1`] = `
Map {
  "Codeship" => Object {
    "scopes": Array [
      "build.read",
      "project.read",
    ],
    "uuid": "dc37c39f-731e-4100-b8bf-69dae9eb14aa",
  },
  "Codeship Test Org" => Object {
    "scopes": Array [
      "build.read",
      "build.write",
      "project.read",
      "project.write",
    ],
    "uuid": "dc37c39f-731e-4100-b8bf-69dae9eb14ab",
  },
}
`;

exports[`authenticate/createOrganizations() should return an empty Map on missing data 1`] = `Map {}`;
